src/containerapp/azext_containerapp/_help.py

```diff
@@ -2131,3 +2131,42 @@
           az containerapp job registry set -n my-containerapp-job -g MyResourceGroup \\
               --server MyContainerappJobRegistry.azurecr.io --identity system-environment
 """
+
+# Maintenance Config Commands
+helps['containerapp env maintenance-config add'] = """
```

Comment:
for naming convention in https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior, if you want to use `add`, then please use `remove` for deletion. if you want to use `delete` for deletion, then please use `create`.